x/protocolpool/README.md

```diff
@@ -4,30 +4,67 @@ sidebar_position: 1
 
 # `x/protocolpool`
 
-Functionality to handle community pool funds. This provides a separate module account for community pool making it easier to track the pool assets. We no longer track community pool assets in distribution module, but instead in this protocolpool module. Funds are migrated from the distribution module's community pool to protocolpool's module account.
-
 ## Concepts
 
-## State
+Protopool is a module that handle functionality around community pool funds. This provides a separate module account for community pool making it easier to track the pool assets. We no longer track community pool assets in distribution module, but instead in this protocolpool module. Funds are migrated from the distribution module's community pool to protocolpool's module account.
 
 ## State Transitions
 
+### FundCommunityPool
+
+FundCommunityPool can be called by any valid account to send funds to the protocolpool module account.
+
+```protobuf
+  // FundCommunityPool defines a method to allow an account to directly
+  // fund the community pool.
+  rpc FundCommunityPool(MsgFundCommunityPool) returns (MsgFundCommunityPoolResponse);
+```
+
+### CommunityPoolSpend
+
+CommunityPoolSpend can be called by the module authority (default governance module account) or any account with authorization to spend funds from the protocolpool module account to a receiver address.
+
+```protobuf
+  // CommunityPoolSpend defines a governance  operation for sending tokens from
+  // the community pool in the x/protocolpool module to another account, which
+  // could be the governance module itself. The authority is defined in the
+  // keeper.
+  rpc CommunityPoolSpend(MsgCommunityPoolSpend) returns (MsgCommunityPoolSpendResponse);
+```
+
 ## Messages
 
-## Begin Block
+### MsgFundCommunityPool
```

Comment:
Can we add that if you know the protocopool address, you can simply use bank send instead.

```md
:::tip
lorem ipsum
::::
```